### PR TITLE
Better travis + tox configuration, version 2

### PR DESCRIPTION
**Pull request by mithrandi**
Updated version of #47.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,28 +1,17 @@
 [tox]
-envlist = py26,py27,pypy,py26-trunkdeps,py27-trunkdeps,pypy-trunkdeps
+envlist = py26-{twlatest,tw150},{py26,py27,pypy}-{twtrunk,twlatest,tw150}
```
Comment (hawkowl) [inline tox.ini]: probably worth making twlatest be 15.4, as 15.5 will break the build

Comment (glyph) [inline tox.ini]: break the 2.6 build? perhaps we should just delete 2.6 support when that happens?

Comment (mithrandi) [inline tox.ini]: Yes, I think in the next Axiom release I will announce that as soon as Twisted releases with 2.6 support dropped, Axiom will be dropping support too.

Comment (hawkowl) [inline tox.ini]: Twisted has (15.3 removed support, 15.4 has unofficial support, 15.5 will refuse to import)

**Pull request merged**